packages/vertica-nodejs/test/integration/gh-issues/2307-tests.js

```diff
@@ -1,3 +1,7 @@
+//Should we remove this file as its a redundant test for TLS mode?
```

Comment:
I think it is reasonable to move this test for bad credentials to tls-tests.js. 

Additionally, I would be partial to not only moving this test into tls-tests.js, but also moving tls-tests.js into mochatest/integration/connection (currently only client directory exists so we would add connection).

This is surely not any sort of requirement for this work so if you would prefer to leave things as they are, that is fine.

Reply:
I can open up another PR later for moving the tests and doing some more cleanup.